applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -605,6 +614,41 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information
 
 .. _matter_bridge_testing:
 
+Configure the functionality of the Matter-Bridge device
```

Comment:
You cannot put it below .. _matter_bridge_testing: because this is a ref tag for the Testing section. It should be above it.